hackerrank/src/Algorithms/Implementation/BetweenTwoSets.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Implementation.BetweenTwoSets;
+
+public class Result
+{
+    public static int getTotalX(List<int> a, List<int> b)
+    {
+        int maxValue = Math.Max(a.Max(), b.Max());
+        var numbers = new List<List<int>>();
+        numbers.Add(new List<int>());
+
+        for (int n = 1; n <= maxValue; n++) // Fill 1,2,3...
+        {
+            numbers[0].Add(n);
+        }
+
+        for (int i = 0; i < a.Count; i++)
+        {
+            numbers.Add(new List<int>());
+            foreach (var item in numbers[i])
```

Comment:
item не чытаецца! Што за item?
```suggestion
            foreach (int denominator in numbers[i])
```
і не хавай тыпы па значэнню (value types) пад `var`, які больш пасуе пад доўгія тыпы па спасылцы (reference types).